shell/browser/api/electron_api_web_frame_main.cc

```diff
@@ -362,8 +362,7 @@ content::FindRequestManager* WebFrameMain::GetOrCreateFindRequestManager() {
   }
   // Concurrent find sessions must not overlap, so destroy any existing
   // FindRequestManagers in any inner WebFrameMain.
-  for (content::RenderFrameHost* rfh :
-       render_frame_->GetMainFrame()->GetFramesInSubtree()) {
+  for (content::RenderFrameHost* rfh : render_frame_->GetFramesInSubtree()) {
```

Comment:
This was added for https://github.com/electron/electron/pull/28274#discussion_r597199100, we can only support concurrent search for frames that are not direct descendants. It should instead be,

```
auto* rfh = render_frame_->GetParent();
while (rfh) {
    auto* web_frame_main = FromRenderFrameHost(rfh);
    if (web_frame_main && web_frame_main->find_request_manager_) {
      web_frame_main->find_request_manager_->StopFinding(
          content::STOP_FIND_ACTION_CLEAR_SELECTION);
      web_frame_main->find_request_manager_.release();
    }
    rfh = rfh->GetParent();
  }
  for (content::RenderFrameHost* rfh : render_frame_->GetFramesInSubtree()) {
    if (rfh == render_frame_)
      continue;
    auto* web_frame_main = FromRenderFrameHost(rfh);
    if (web_frame_main && web_frame_main->find_request_manager_) {
      web_frame_main->find_request_manager_->StopFinding(
          content::STOP_FIND_ACTION_CLEAR_SELECTION);
      web_frame_main->find_request_manager_.release();
    }
  }
```

Reply:
Added this in and ran the tests locally again to confirm they still pass.